skops/card/_model_card.py

```diff
@@ -172,8 +210,11 @@ class Card:
 
     Parameters
     ----------
-    model: estimator object
-        Model that will be documented.
+    model: pathlib.path, str, or sklearn estimator object
+        ``Path``/``str`` of the model or the actual model instance that will be
+        documented. If a ``Path`` or ``str`` is provided, model will be loaded
+        on first use.
```

Comment:
```suggestion
        documented. If a ``Path`` or ``str`` is provided, model will be loaded.
```

After the change, it will be loaded each time, so "on first use" is not correct anymore.